client/state/action-types.js

```diff
@@ -78,6 +78,8 @@ export const COMMENTS_REPLY_WRITE = 'COMMENTS_REPLY_WRITE';
 export const COMMENTS_REQUEST = 'COMMENTS_REQUEST';
 export const COMMENTS_REQUEST_FAILURE = 'COMMENTS_REQUEST_FAILURE';
 export const COMMENTS_REQUEST_SUCCESS = 'COMMENTS_REQUEST_SUCCESS';
+export const COMMENTS_TREE_RECEIVE = 'COMMENTS_TREE_RECEIVE';
+export const COMMENTS_TREE_REQUEST = 'COMMENTS_TREE_REQUEST';
```

Comment:
`COMMENTS_TREE_SITE_REQUEST ` maybe, since we might have a tree for a posts or even all sites.